### PR TITLE
Move everything out of Known Issues and into more appropriate locations.

### PR DESCRIPTION
**Pull request by hramos**
Two of the known issues have been moved to the issue tracker:

* #8315 
* #8316 

Others have been moved into more appropriate locations, such as the `TextInput` issue to the API doc itself, and the React debugging issue to the Debugging doc.

The Android-specific compatibility concerns have been dropped entirely as it does not seem like people would find these in the docs.

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @andrewsardone and @caabernathy to be potential reviewers.

File: docs/KnownIssues.md [hunk 1]
```diff
@@ -1,90 +0,0 @@
...
-
-It's [currently not possible](https://github.com/facebook/react-devtools/issues/229) to use the "React" tab in the devtools to inspect app widgets. This is due to a change in how the application scripts are evaluated in the devtools plugin; they are now run inside a Web Worker, and the plugin is unaware of this and so unable to communicate properly with React Native.
-
-However, you can still use the Console feature of the devtools, and debugging JavaScript with breakpoints works too. To use the console, make sure to select the `⚙debuggerWorker.js` entry in the devtools dropdown that by default is set to `<top frame>`.
-
-### Missing Android Modules and Views
-
-The work on React Native for Android started later than React Native for iOS. Most views and modules are now available on Android, with the following exceptions:
-
-#### Views
```
Comment (JoelMarcey) [inline docs/KnownIssues.md]: I noticed that some of these were neither moved or made into issues (I don't think). Do just feel like they are not needed any longer as well?

Comment (hramos) [inline docs/KnownIssues.md]: I dropped some of these as it did not seem like it was worth calling out in the docs. Many seemed like, if they are still an issue, they would be easily discovered by performing a search for the issue.

Happy to reinstate them, it just seems like there's a grab bag of random issues in this doc that may no longer be relevant.

Comment (JoelMarcey) [inline docs/KnownIssues.md]: Nope. I am good. Shipping it.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1702465040004696/int_phab) to review.

**Pull request closed without merging**